src/scene/particle-system/particle-emitter.js

```diff
@@ -820,6 +824,10 @@ Object.assign(pc, function () {
                     fog: (this.emitter.scene && !this.emitter.noFog) ? this.emitter.scene.fog : "none",
                     wrap: this.emitter.wrap && this.emitter.wrapBounds,
                     localSpace: this.emitter.localSpace,
+
+                    // in Editor, screen space particles (children of 2D Screen) are still rendered in 3d space
+                    screenSpace: inTools ? false : this.emitter.screenSpace,
```

Comment:
I think this should be set by the Editor (like how the Editor makes the screen a 3D screen and not the engine). We made an effort in the past to get rid of code like this which is basically editor code sprinkled in the engine.

Reply:
done in the linked PR, closing this one due to complexity merging in rollup changes.